testkit/src/main/scala/org/bitcoins/testkit/fixtures/BitcoinSFixture.scala

```diff
@@ -36,6 +36,9 @@ trait BitcoinSFixture extends BitcoinSAsyncFixtureTest {
       test(fixture.asInstanceOf[FixtureParam]).toFuture
     }
 
+    outcomeF.failed.foreach(err =>
+      println(s"Failed fixture test execution=${err}"))
```

Comment:
nit: `println -> logger.error`

Reply:
I'm going to keep this now to help with debugging on CI as we have logging turned off there